services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -358,7 +365,7 @@ private void processSlot() {
         EVENT_LOG.epochEvent();
       }
 
-      this.eventBus.post(new SlotEvent(nodeSlot));
+      eventChannels.getPublisher(SlotEventsChannel.class).onSlot(new SlotEvent(nodeSlot));
```

Comment:
nit: Would be neater if we got the publisher once during init and then could just call it directly here instead of having to call `getPublisher` each time (no performance impact, just clearer to read).

Reply:
Done.

Reply:
Done.